todo/utils/formatting.py

```diff
@@ -2,13 +2,14 @@
 # Licensed under MIT
 
 from typing import List
+import logging
 
 __all__ = ["_format_todos", "_format_completed", "_build_underline"]
 
+log = logging.getLogger("red.jojocogs.todo.formatting")
```

Comment:
```suggestion

__all__ = ["_format_todos", "_format_completed", "_build_underline"]

```
This variable isn't used here.